test/e2e/case38_install_operator_test.go

```diff
@@ -170,13 +170,32 @@ var _ = Describe("Testing OperatorPolicy", Ordered, func() {
 		ConsistentlyWithOffset(1, checkFunc, consistentlyDuration, 1).Should(Succeed())
 	}
 
+	preFunc := func() {
+		if IsHosted {
+			KubectlTarget("create", "ns", opPolTestNS)
+			DeferCleanup(func() {
+				KubectlTarget("delete", "ns", opPolTestNS)
+			})
+		}
+		utils.Kubectl("create", "ns", opPolTestNS)
+		DeferCleanup(func() {
+			utils.Kubectl("delete", "ns", opPolTestNS)
+		})
```

Comment:
Should this be in an `else`?

Reply:
I use this clean policy in manage cluster(not hosted cluater)